docs/trex_release-notes.md

```diff
@@ -8,12 +8,18 @@ layout: docs
 * TOC
 {:toc}
 
+### Tableau 2018.2 (Release Candidate)
+*July, 2018*
+
+Coming soon.
+ 
+
 ### Tableau 2018.2 (Beta 2)
 *June 4, 2018*
 
 Update of the Tableau Extensions API
 
-- Tableau Extensions API library: tableau-extensions-0.16.0.js
+- Tableau Extensions API library: `tableau-extensions-0.16.0.js`
```

Comment:
We are moving to version 1.0.0 this coming week.

Reply:
Yes. The 2018.2 release will say 1.0.   We just put the latest release on the top of the page. 
The release notes for Beta 2 on June 4 point to the 0.16 library and Tableau Beta 2 downloads.